### ISSUE TITLE
mkdir on Linux Mint 17.3 hangs

### ISSUE THREAD

**Issue by sdigit** [keybase/client#2662]
Was playing with KBFS; ran into an issue when running mkdir /keybase/public/dive/Photos: it hung, and kbfs now hangs when I do anything (so it feels like when an NFS server dies out from under you)

The following was in `/keybase/kbfs.error.txt`
```
md-server: Unhandled incoming RPC
```

my log id: 5d2931c5218d4b9406cfc51c

**Comment by cjb** [comment]
@strib Please take a look.

**Comment by cjb** [comment]
Oh wait, I bet this is the bug that @patrickxb is working on.  It hasn't made it into a public OS X build as far as we know, but it's in Linux builds.

**Comment by sdigit** [comment]
Also noticed: https://keybase.io/pub/dive returns a 404 (whether or not i end it with a /); not surprising just thought the datapoint might help.

**Comment by cjb** [comment]
@sdigit That'd be https://keybase.pub/dive/.

**Comment by sdigit** [comment]
Yeah I just noticed that, and that address works just fine. Oops

**Comment by cjb** [comment]
Thanks for the report, we're fixing this at the moment, I'd expect tomorrow's Linux package will contain the fix.

**Comment by cjb** [comment]
By the way, we think that running `keybase unlock` will get KBFS alive again for you as a workaround.

**Comment by sdigit** [comment]
Yep, looks that way -- i was able to make the directory and copy a couple photos into it just fine.
I did `umount /keybase` and `run_keybase` again after `keybase unlock`, as it didn't change the pre-unlock mount (but I'm guessing it wouldn't have, as it was mounted before being unlocked)
